routers/repo/http.go

```diff
@@ -364,14 +364,23 @@ func serviceRPC(h serviceHandler, service string) {
 	}
 
 	if h.cfg.OnSucceed != nil {
-		input, err = ioutil.ReadAll(reqBody)
+		tmpfile, err := ioutil.TempFile("", "gogs")
```

Comment:
I hadn't found info about the permissions that TempFile would put on the file, I guess it should be checked